content/en/docs/components/katib/user-guides/metrics-collector.md

```diff
@@ -80,3 +84,64 @@ To define the metrics collector for your Experiment:
    recall=0.55
    precision=.5
    ```
+
+## Push-based Metrics Collector
+
+Your training code needs to call [`report_metrics`](https://github.com/kubeflow/katib/blob/master/sdk/python/v1beta1/kubeflow/katib/api/report_metrics.py#L26) function in Python SDK to record metrics.
+
+To define the push-based metrics collector for your Experiment, you have two options:
+
+- YAML File
+
+    1. Specify the collector type `Push` in the `.collector.kind` field.
+
+    2. Write code in your training container to call `report_metrics` to report metrics.
+
+- [`tune`](https://github.com/kubeflow/katib/blob/master/sdk/python/v1beta1/kubeflow/katib/api/katib_client.py#L166) function
+
+    Use tune function and specify the `metrics_collector_config` field. You can reference to the following example:
+
+    ```
+    import kubeflow.katib as katib
```

Comment:
Let's use simple example here and remove all unnecessary parameters and function calls.

Reply:
@andreyvelich Sorry, I may not fully understand what you mean "unnecessary parameters and function calls".

I copied the example in the [get-started](https://www.kubeflow.org/docs/components/katib/getting-started/#getting-started-with-katib-python-sdk) chapter and replaced `print` with `report_metrics()`. Do you mean that I don't need to call the `tune()` function?

Reply:
I think, we should just show example how user can use `tune` API with `push-based` metrics collector without any additional parameters they should set. For example:
```py
import kubeflow.katib as katib

def objective(parameters):
  import time
  import kubeflow.katib as katib
  time.sleep(5)
  result = 4 * int(parameters["a"])
  # Push metrics to Katib DB.
  katib.report_metrics({"result": result})

katib.KatibClient().tune(
  name="push-metrics-exp",
  objective=objective,
  parameters= {"a": katib.search.int(min=10, max=20)}
  objective_metric_name="result",
  max_trial_count=2,
  metrics_collector_config={"kind": "Push"},
  # When SDK is released, replace it with packages_to_install=["kubeflow-katib==0.18.0"] 
  packages_to_install=["git+https://github.com/kubeflow/katib.git@master#subdirectory=sdk/python/v1beta1"],
)
```

That should allow user to focus on important changes they need to make to try this out.

Reply:
@andreyvelich Thanks for your clarification! I'll update the blog.